src/classes/os-utils.ts

```diff
@@ -1,6 +1,6 @@
 import { CommandObj, UserConfigJson, OperatingSystems } from "../interfaces/types";
 import { OsCommands } from "./os-commands";
-import fs from "fs";
+import { userConfigUtils } from "./user-config-utils";
```

Comment:
Class names should be captitalized

```suggestion
import { UserConfigUtils } from "./user-config-utils";
```